examples/pkg/apis/example/v1beta1/types.go

```diff
@@ -19,7 +19,6 @@ package v1beta1
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 // +genclient
-// +genclient:skipVerbs=list,watch
```

Comment:
This is now somehow broken after 1.31 migrated to generics. And I'm missing historical background why this was here in the first case?

Reply:
I might be wrong but looks like regression in code-generator and not our code

Reply:
I generate clients wrong without DeleteCollections methods.

Reply:
you mean they are missing?

Reply:
Fixed. I think its getting just further from upstream :)

Reply:
upstream client changed. Basically this skips List,Watch and DeleteCollections. And this breaks our generator a  bit.
I think we still need to refactor this to v2 generator. Not sure if it makes sense to invest now :/